src/types/encoded_data.rs

```diff
@@ -10,12 +10,14 @@ use scylla::{
 
 use crate::errors::js_error;
 
+#[derive(Clone)]
 enum MaybeUnsetNullableValue<T> {
     Value(T),
     Null,
     Unset,
 }
 
+#[derive(Clone)]
 pub struct EncodedValuesWrapper {
     inner: MaybeUnsetNullableValue<Vec<u8>>,
 }
```

Comment:
:question: Why is this needed? Deriving `Clone` for these items hints me that something's not OK. We shouldn't clone these items for each page, rather borrow them, right?